packages/feedback-service/src/typedef.graphql

```diff
@@ -1,5 +1,37 @@
 scalar DateTime
 
+type createdBy {
```

Comment:
Just create a single type called UserType and use it for both `createdBy` and `updatedBy`

```suggestion
type UserType {
```